mempool/bench_test.go

```diff
@@ -36,7 +38,8 @@ func init() {
 	txs = types.Txs{}
 	for i := 0; i < txSize; i++ {
 		txBytes := make([]byte, 20)
-		priority := strconv.FormatInt(int64(i)%100, 10)
+		randInt, _ := rand.Int(rand.Reader, big.NewInt(100))
+		priority := strconv.FormatInt(randInt.Int64(), 10)
```

Comment:
modulo 100 to make it more realistic?

Reply:
`func Int(rand io.Reader, max *big.Int) `
rand.Int returns a uniform random value in [0, max)